src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -114,23 +144,166 @@ const ReservationList: FC<ReservationListProps> = ({ pageSize }) => {
     !isLoadingFBS &&
     !isLoadingPublizon;
 
+  const openReservationDeleteModal = (deleteId: string | number | null) => {
+    setModalDeleteId(deleteId);
+    open(`${deleteReservation}${deleteId}`);
+  };
+
+  const openReservationDetailsModal = useCallback(
+    (reservationInput: ReservationType) => {
+      setReservation(reservationInput);
+      open(
+        `${reservationDetails}${
+          reservationInput.faust || reservationInput.identifier
+        }`
+      );
+    },
+    [open, reservationDetails]
+  );
+
+  const findReservationInLists = useCallback(
```

Comment:
You could consider making a custom hook to make the component shorter in order to improve the overview of the code